common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1958,16 +1958,24 @@ def ensure_indexes(self):
                 ('_id.name', pymongo.ASCENDING),
                 ('_id.revision', pymongo.ASCENDING),
             ],
-            background=True)
+            # While this index would more safely be "background=True", it's not currently created that way.
```

Comment:
That is both correct - and not ideal. These `create_index` commands raise an exception if any of the (valid) options are different. In this case, edx.org created this index with `background=True` - therefore causing the exception. Another option is to have `background=True` and to re-create the edx.org index - which means dropping the index and creating it again.